### PR TITLE
[WA-44] Fix env var syntax in PowerShell commands

### PR DESCRIPTION
**Pull request by clarkb7**
### What does this PR do?

Fixes the environment variable syntax in PowerShell commands. `%ProgramFiles%` is for `cmd.exe`. `$env:ProgramFiles` is for `powershell.exe`.

Normalizes the case of the environment variables for consistency between pages and commands, for example `PROGRAMFILES` -> `ProgramFiles`. Though env vars are case insensitive on Windows the latter case matches what is reported by the `env` command in `cmd.exe` and `dir env:` in `powershell.exe`.

Normalizes "PowerShell" case (https://docs.microsoft.com/en-us/powershell/scripting/overview?view=powershell-7.2)

### Motivation

PowerShell throws an error if you try to use env vars like `%ProgramFiles%`. The proper syntax is `$env:ProgramFiles`.

xref [WA-44](https://datadoghq.atlassian.net/browse/WA-44)

### Additional Notes

I think these commands should be updated in the other languages, too, but I can't read them to know for sure.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by alai97** [comment]
Created https://datadoghq.atlassian.net/browse/DOCS-3958 for validation!

File: content/en/agent/faq/log4j_mitigation.md [hunk 1]
```diff
@@ -152,10 +152,10 @@ $stream.Close()
 $stream.Dispose()
 ```
 
-Stop the Datadog Agent service before applying the patch:
+From an **elevated(run as Admin)** PowerShell, stop the Datadog Agent service before applying the patch:
```
Comment (cswatt) [inline content/en/agent/faq/log4j_mitigation.md:155]:
```suggestion
From an **elevated** (run as admin) PowerShell, stop the Datadog Agent service before applying the patch:
```

File: content/en/agent/guide/integration-management.md [hunk 2]
```diff
@@ -43,17 +43,19 @@ The syntax for this command is `datadog-agent integration install <INTEGRATION_P
...
+{{% tab "Linux" %}}
 ```shell
 sudo -u dd-agent -- datadog-agent integration install datadog-vsphere==3.6.0
 ```
-
-Windows:
-
+{{% /tab %}}
+{{% tab "Windows PowerShell" %}}
+Run `powershell.exe` as **elevated(run as Admin)**
```
Comment (cswatt) [inline content/en/agent/guide/integration-management.md:53]:
```suggestion
Run `powershell.exe` as **elevated** (run as admin).
```

File: content/en/agent/guide/integration-management.md [hunk 3]
```diff
@@ -73,17 +75,19 @@ To remove an integration, use the `datadog-agent integration remove` command. Th
...
+{{% tab "Linux" %}}
 ```shell
 sudo -u dd-agent -- datadog-agent integration remove datadog-vsphere
 ```
-
-Windows:
-
+{{% /tab %}}
+{{% tab "Windows PowerShell" %}}
+Run `powershell.exe` as **elevated(run as Admin)**
```
Comment (cswatt) [inline content/en/agent/guide/integration-management.md:85]:
```suggestion
Run `powershell.exe` as **elevated** (run as admin).
```

File: content/en/agent/guide/integration-management.md [hunk 4]
```diff
@@ -93,33 +97,37 @@ To get information, such as the version, about an installed integration, use the
...
+{{% tab "Linux" %}}
 ```shell
 sudo -u dd-agent -- datadog-agent integration show datadog-vsphere
 ```
-
-Windows:
-
+{{% /tab %}}
+{{% tab "Windows PowerShell" %}}
+Run `powershell.exe` as **elevated(run as Admin)**
```
Comment (cswatt) [inline content/en/agent/guide/integration-management.md:107]:
```suggestion
Run `powershell.exe` as **elevated** (run as admin).
```

File: content/en/agent/guide/integration-management.md [hunk 5]
```diff
@@ -93,33 +97,37 @@ To get information, such as the version, about an installed integration, use the
...
+{{% tab "Linux" %}}
 ```text
 sudo -u dd-agent -- datadog-agent integration freeze
 ```
-
-Windows:
-
+{{% /tab %}}
+{{% tab "Windows PowerShell" %}}
+Run `powershell.exe` as **elevated(run as Admin)**
```
Comment (cswatt) [inline content/en/agent/guide/integration-management.md:125]:
```suggestion
Run `powershell.exe` as **elevated** (run as admin).
```

File: content/en/agent/troubleshooting/agent_check_status.md [hunk 6]
```diff
@@ -63,18 +63,17 @@ If your issue continues, [reach out to the Datadog support team][1] with a [flar
 {{< tabs >}}
 {{% tab "Agent v6 & v7" %}}
 
-Run the following script, with the proper `<CHECK_NAME>`:
+Run the following script from an *elevated(run as Admin)* PowerShell command line, with the proper `<CHECK_NAME>`:
```
Comment (cswatt) [inline content/en/agent/troubleshooting/agent_check_status.md:66]:
```suggestion
Run the following script from an **elevated** (run as admin) PowerShell command line, with the proper `<CHECK_NAME>`:
```

File: content/en/agent/troubleshooting/agent_check_status.md [hunk 7]
```diff
@@ -92,14 +91,14 @@ This outputs any metrics or events that the check returns.
 {{% /tab %}}
 {{% tab "Agent v>=5.12" %}}
 
-Run the following script, with the proper `<CHECK_NAME>`:
+Run the following script from an *elevated(run as Admin)* PowerShell command line, with the proper `<CHECK_NAME>`:
```
Comment (cswatt) [inline content/en/agent/troubleshooting/agent_check_status.md:94]:
```suggestion
Run the following script from an **elevated** (run as admin) PowerShell command line, with the proper `<CHECK_NAME>`:
```

File: content/en/developers/guide/custom-python-package.md [hunk 8]
```diff
@@ -42,24 +42,24 @@ sudo /opt/datadog-agent/embedded/bin/pip install <PACKAGE_NAME>
 
 {{% tab "Windows" %}}
 
-Custom Python packages can be installed using the Agent's embedded Python using the following command in PowerShell:
+Custom Python packages can be installed using the Agent's embedded Python using the following command in an **elevated(run as Admin)** PowerShell command line:
```
Comment (cswatt) [inline content/en/developers/guide/custom-python-package.md:45]:
```suggestion
Custom Python packages can be installed using the Agent's embedded Python using the following command in an **elevated** (run as admin) PowerShell command line:
```

**Review by cswatt — COMMENTED** [review]
small nits

**Review by cswatt — APPROVED** [review]

**Pull request merged**